Makefile

```diff
@@ -175,6 +175,14 @@ push-bench-images: install-bench
 	docker tag pachyderm_test pachyderm/bench:`git rev-list HEAD --max-count=1`
 	docker push pachyderm/bench:`git rev-list HEAD --max-count=1`
 
+tag-images:
+	docker tag pachyderm_pachd pachyderm/pachd:`$(GOPATH)/bin/pachctl version 2>/dev/null | grep pachctl | awk -v N=2 '{print $$N}'`
+	docker tag pachyderm_worker pachyderm/worker:`$(GOPATH)/bin/pachctl version 2>/dev/null | grep pachctl | awk -v N=2 '{print $$N}'`
+
+push-images: tag-images
+	docker push pachyderm/pachd:`$(GOPATH)/bin/pachctl version 2>/dev/null | grep pachctl | awk -v N=2 '{print $$N}'`
+	docker push pachyderm/worker:`$(GOPATH)/bin/pachctl version 2>/dev/null | grep pachctl | awk -v N=2 '{print $$N}'`
+
```

Comment:
Why are we adding these targets? Our `Makefile` already tags and pushes images doesn't it? I'd prefer not to add a second way to do that if it's possible to use / extend the existing way. Also both of these commands are using `pachctl` but don't list `install` the target that installs `pachctl` as a dependency.